infrastructure/lambda-logout.tf

```diff
@@ -0,0 +1,87 @@
+module "logout-gateway" {
+  # Gateway Variables
+  source              = "./modules/gateway"
+  api_gateway_id      = aws_api_gateway_rest_api.ndr_doc_store_api.id
+  parent_id           = aws_api_gateway_resource.auth_resource.id
+  http_method         = "GET"
+  authorization       = "NONE"
+  gateway_path        = "Logout"
+  require_credentials = false
+  origin              = "'https://${terraform.workspace}.${var.domain}'"
+  # Lambda Variables
+  api_execution_arn = aws_api_gateway_rest_api.ndr_doc_store_api.execution_arn
+  owner             = var.owner
+  environment       = var.environment
+
+  depends_on = [
+    aws_api_gateway_rest_api.ndr_doc_store_api,
+  ]
+}
+
```

Comment:
Do we need the pre-flight for the logout?

Reply:
Yes, as this is an in-browser axios request. You get a cors error because then the endpoint doesn't know how to function.

In ARF we did not need preflight on logout because it used to be another redirect